strax/chunk.py

```diff
@@ -219,7 +220,17 @@ def split(self, t: ty.Union[int, None], allow_early_split=False):
             target_size_mb=self.target_size_mb,
         )
 
-        subruns_first_chunk, subruns_second_chunk = _split_runs_in_chunk(self.subruns, t)
+        if self.is_superrun and (
```

Comment:
In normal cases, the time interval of superrun exactly covers the data. If it does not, it means we might be resetting the start and end time in metadata.